docker/env/version

```diff
@@ -1 +1 @@
-0.38
+0.39
```

Comment:
I think when testing new versions we should use some testing version, like 0.39a, 0.39b,...0.39z

Reply:
next time I'll try to remember that...